sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -410,7 +501,7 @@ def _client_ready(self):
         states.
 
         :rtype: bool
-        :raises: ~uamqp.errors.MessageHandlerError if the MessageReceiver
```

Comment:
This is interesting because we don't have a MessageHandlerError like uamqp does, nor were we raising an error if the link state was on error

Reply:
Correct - MessageHandler is no longer an entity - either the Link is in error state, or a Message on the link has failed to send/receive/settle. We no longer have a stateful middle layer between them.